### PR TITLE
Code Review PA 1

### PR DESCRIPTION
**Pull request by lindseyreynolds44**

### CODE REVIEW

File: cst438-homework1/src/main/java/cst438/MovieRating.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+  @Id
+  @GeneratedValue
+  private long id;
+
+  @NotNull
+  @Size(min = 3, max = 25)
+  private String title;
+
+  @NotNull
```
Comment (thedandano) [inline cst438-homework1/src/main/java/cst438/MovieRating.java:25]: I would recommend adding a minimum and maximum constraints to the variable that stores the rating. 
@min(1)
@max(5)
See the link below for further documentation on constraints.
https://docs.oracle.com/javaee/7/tutorial/bean-validation001.htm

File: cst438-homework1/src/main/resources/templates/movie_form.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+<title>Movie Form</title>
+</head>
+<body>
+	<h2>Rate a Movie!</h2>
+	<form action="#" th:action="@{/movies/new}" th:object="${movieRating}" method="post">
+	Reviewer's name: <input type="text" th:field="*{reviewerName}" /> 
+		<span th:if="${#fields.hasErrors('reviewerName')}" th:errors="*{reviewerName}"> </span> <br />
+	Movie Title: <input type="text" th:field="*{title}" />
+	    <span th:if="${#fields.hasErrors('title')}" th:errors="*{title}"> </span> <br /> 
+	Rating (1-5): <input type="number" min="1" max="5" th:field="*{rating}" />
```
Comment (thedandano) [inline cst438-homework1/src/main/resources/templates/movie_form.html:14]: This is min and max handling is good here, but don't forget to also add validation on the backend.

**Review by thedandano — CHANGES_REQUESTED** [review]
Overall great job! It just needs input validation on the backend.

File: cst438-homework1/src/main/java/cst438/MovieRating.java (hunk 1 above)
Comment (lindseyreynolds44) [reply cst438-homework1/src/main/java/cst438/MovieRating.java:25]: I decided to put constraints on the front end instead, but its probably smart to cover all bases and do both!

**Pull request merged**